documentation/content/101 Quick Start/updating-new-releases.md

```diff
@@ -0,0 +1,133 @@
+---
+Title: Updating Elegant To the Latest Release
+Tags: pelican-theme, new-releases
+Category: 101 — Quick Start
+Date: 2019-12-06
+Slug: updating-elegant-to-the-latest-release
+Comment_id: qryv67o-updating-elegant-to-the-latest-release
+authors: Jack De Winter
+---
+
+Once you have determined that you want to update to the latest release of Elegant, the
+process is usually quite simple. There are wo main ways to get the Elegant theme: as
+a standalone repository and as part of the Pelican-Themes repository.
+
+## Via a Standalone Repository
+
+If you installed the Elegant theme as a standalone repository, you most likely followed
+directions like the following to clone the Elegant repository:
+
+```bash
+mkdir ../blog-theme/Elegant
+git clone https://github.com/Pelican-Elegant/elegant ../blog-theme/Elegant
+```
+
+For this example, the assumption is that the Pelican directory where your website is
+stored is at the same level as the `blog-theme` directory. As such, the `../blog-theme`
+part of the path gets us to the theme part of the directory tree. It is possible to just
+leave the path at that, but it is recommended to add the `/Elegant` at the end as a
+reminder of the repository name.
+
+If the Elegant theme was installed in this manner, you can either update your local
+repository or recreate your repository as noted in the following two sections.
+
+### Updating a Standalone Theme
+
+The `master` branch of Elegant's Git repository always contains the latest release. To
+update to that release, go into the Elegant theme directory (`../blog-theme/Elegant`
+from the example above), and enter the following command.
+
+```bash
+git pull origin master
+```
+
+Note that this is the most effective way of updating your standalone repository if you
+have not made any changes to the the theme. If you have made some changes, you may be
+required to merge the changes in your local repository with any changes in the release
+version you pulled down with this command.
+
+### Recreate a Standalone Theme
+
+If you are lucky, you will not reach a time where you believe, even a tiny bit, that you
+have messed up the contents of your theme repository. If you have made any changes to the
+base Elegant theme, it is inevitable that at some point you will need to reset the
+repository to a known good state. Assuming that the repository is in the directory
+`../blog-theme/Elegant`, per the example above, executing the following command:
+
+```bash
+rmdir -rf ../blog-theme/Elegant
+```
+
+will remove the entire theme from your local system. Once this is done, you can follow
+the clone directions in the example at the top of this section on standalone
+repositories to restore the Elegant theme directory to the same state as Elegant's
+master branch on GitHub.
+
+## Via the Pelican-Themes Repository
```

Comment:
Move this section to the top. Most first time users would clone pelican themes repository.

Also, the instruction should be

1. Clone repo
2. cd into the folder
2. Run `git submodule update --init elegant`
3. Run  `git submodule update --remote elegant`

Reply:
moved. and changed instructions to match.